pkgs/by-name/wa/wamr/package.nix

```diff
@@ -7,32 +7,43 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "wamr";
-  version = "2.2.0";
+  version = "2.3.0";
 
   src = fetchFromGitHub {
     owner = "bytecodealliance";
     repo = "wasm-micro-runtime";
     rev = "WAMR-${finalAttrs.version}";
-    hash = "sha256-Rhn26TRyjkR30+zyosfooOGjhvG+ztYtJVQlRfzWEFo=";
+    hash = "sha256-Qz1Onig++J4V2dpse6n8WADEuJPqTpXK5B+fQml8C2E=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = lib.optionals stdenv.hostPlatform.isDarwin [
+  cmakeFlags = [
+    "-DWAMR_BUILD_SIMD=0"
```

Comment:
Going to see how feasible enabling SIMD is, given we package simde already it shouldn't be hard. Leaving this here so I remember.